manage.py

```diff
@@ -2,62 +2,75 @@
 """
 CLI to manage redash.
 """
+import code
```

Comment:
Oops, that's when I was trying to add a `shell` command and not noticing flask already provided one.